components/engine/engine-odata/src/main/java/org/eclipse/dirigible/components/odata/transformers/ODataDatabaseMetadataUtil.java

```diff
@@ -204,10 +204,10 @@ private void getForeignKeys(DatabaseMetaData databaseMetadata, Connection connec
 
 
         while (foreignKeys.next()) {
-            TableConstraintForeignKey relationMetadata = new TableConstraintForeignKey(foreignKeys.getString(JDBC_FK_NAME_PROPERTY),
-                    new String[] {}, new String[] {foreignKeys.getString(JDBC_FK_COLUMN_NAME_PROPERTY)},
-                    foreignKeys.getString(JDBC_PK_TABLE_NAME_PROPERTY), foreignKeys.getString(JDBC_PK_SCHEMA_NAME_PROPERTY),
-                    new String[] {foreignKeys.getString(JDBC_PK_COLUMN_NAME_PROPERTY)}, tableMetadata.getConstraints());
+            new TableConstraintForeignKey(foreignKeys.getString(JDBC_FK_NAME_PROPERTY), new String[] {},
```

Comment:
Does `new TableConstraintForeignKey(...)` have a side effect? As far as I saw, it uses the constraints from the `tableMetadata`

Reply:
fixed